lib/ruby_lexer.rb

```diff
@@ -22,6 +22,7 @@ class RubyLexer
   STR_FUNC_QWORDS = 0x08
   STR_FUNC_SYMBOL = 0x10
   STR_FUNC_INDENT = 0x20 # <<-HEREDOC
+  STR_FUNC_ICNTNT = 0x40 # <<~HEREDOC
```

Comment:
`indented content` I tried to follow the fixed width constant names, but certainly not ideal.